manuscript.Rmd

```diff
@@ -161,9 +162,17 @@ It might well be the case that psychology needs more confirmatory studies to pro
 However, independently of such a goal, preregistration can be useful for any kind of study on the continuum between strictly confirmatory and fully exploratory.
 
 <!---outline-structure--->
-To form such an objective for preregistration, we first introduce some tools of Bayesian philosophy of science and map the exploration/confirmation distinction onto a dimensional quantity we call "theoretical risk" [a term borrowed from @meehlTheoreticalRisksTabular1978, but formalized as the probability of proving a hypothesis wrong if it does not hold], which is inversely related to the type-I error rate in null hypothesis testing.
+To form such an objective for preregistration, we first introduce some tools of Bayesian philosophy of science and map the exploration/confirmation distinction onto a dimensional quantity we call "theoretical risk" [a term borrowed from @meehlTheoreticalRisksTabular1978, but formalized as the probability of proving a hypothesis wrong if it does not hold].
 
-Further, we outline two interpretations of preregistration.
+We are interested in why preregistrations should change researchers' evaluation of evidence.
+Applying a Bayesian framework allows us to investigate our research question most straightforwardly. 
+Specifically, it allows us to model changes in subjective degrees of belief due to preregistration or, more simply, "persuasion".
+Please note that our decision to adopt a Bayesian philosophy of science does not make assumptions about the statistical methods researchers use.
+In fact, this conceptualization is intentionally as minimal as possible to be compatible with a wide range of philosophies of science and statistical methods researchers might subscribe to.
+However, we should note that Popperians would be appalled that we are content with positive inductive inferences (but we dislike "failing to disprove" too much to give it up), and Neopopperians would flinch that we assign probabilities to beliefs (we are fond of calculating things).
```

Comment:
I don't understand "we dislike it too much to give it up". I would understand if you say "we _like_ it too much to give it up" or the opposite ("we dislike it too much to keep it").

Reply:
True!

```suggestion
However, we should note that Popperians would be appalled that we are content with positive inductive inferences (but we dislike "failing to disprove" too much), and Neopopperians would flinch that we assign probabilities to beliefs (we are fond of calculating things).
```

Reply:
```suggestion
However, we should note that Popperians would be appalled that we are content with positive inductive inferences (but we regard "failing to disprove" as too limited), and Neopopperians would flinch that we assign probabilities to beliefs (we are fond of calculating things).
```